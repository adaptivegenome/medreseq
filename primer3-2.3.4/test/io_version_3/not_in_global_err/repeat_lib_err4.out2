../src/primer3_core: Sequence length < 3 in mispriming library repeat_lib4
