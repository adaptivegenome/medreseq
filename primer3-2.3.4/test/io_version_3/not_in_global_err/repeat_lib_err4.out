PRIMER_SEQUENCE_ID=seq1
SEQUENCE=ATGCTAGCTAGTCGATCGATGCTAGCTAGTCGATCTACTATCATCATGCTAGCATGGGGCATCTGTGGGATGCTGTCGTACTGATCGATGCTAGCTAGCTAGTCGATCGTAGCTATCATCATCTAGTCATCGTAGCTATCATCGTAGC
PRIMER_MISPRIMING_LIBRARY=repeat_lib4
PRIMER_ERROR=Sequence length < 3 in mispriming library repeat_lib4
=
